shotover-proxy/src/transforms/distributed/consistent_scatter.rs

```diff
@@ -170,32 +151,24 @@ impl Transform for ConsistentScatter {
         drop(rec_fu);
 
         Ok(if results.len() < max_required_successes as usize {
-            required_successes
-                .iter()
-                .map(|_| {
-                    Message::new_response(
-                        QueryResponse::empty_with_error(Some(MessageValue::Strings(
-                            "Not enough responses".to_string(),
-                        ))),
-                        true,
-                        Frame::None,
-                    )
-                })
-                .collect()
+            let mut messages = message_wrapper.messages;
+            for message in &mut messages {
+                message.original = Frame::Redis(RedisFrame::Error(
+                    Str::from_inner("Not enough responses".into()).unwrap(),
```

Comment:
Can we implement Message::set_error either in this PR or in a prereq PR so that we dont have this code hardcoded to a redis error?

Reply:
I'll open a pre-req PR.

Reply:
Wasn't able to decouple the changes in consistent scatter so I have included it in this PR.